aptos-move/block-executor/src/view.rs

```diff
@@ -1048,7 +1102,7 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>> LatestView<'a, T, S> {
                 true
             },
             // TODO: store incorrect use in UnsyncMap and eliminate this API.
```

Comment:
What happens with this todo now?

Reply:
nice find, we resolve it! (it didn't need to be in UnsyncMap, UnsyncReadSet also works, and is actually more consistent w. parallel / capturedreads)